vcd/resource_vcd_rde_type_behavior_and_acl_test.go

```diff
@@ -199,10 +225,20 @@ resource "vcd_rde_interface_behavior" "behavior2" {
   rde_interface_id = vcd_rde_interface.interface.id
   name             = "{{.Name}}2"
   description      = "{{.Description}}"
-  execution = {
```

Comment:
Curious, why previous way is removed from the test, we don't want to cover it anymore?

Reply:
There's still the `vcd_rde_interface_behavior.behavior1` test resource which uses the "old" argument, but it doesn't show as it was not modified.